test/symlink.test.js

```diff
@@ -97,6 +97,23 @@ test('[symlink] fails to create symlink - directory not found', function(assert)
   });
 });
 
+test('[symlink] overwrites existing files', function(assert) {
+  var src = path.join(__dirname + '/fixtures/', 'config.cpp');
+  var dst = path.join(__dirname + '/fixtures/', 'config-copy.cpp');
```

Comment:
How about using much more simple files - like text files with a single word in them? This would make the test more minimal increasing signal to noise (since protozero code specifics not matter)